ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2400,38 +2412,33 @@ private enum UnityLayers {
     RemoveBuggyTidalLocking();
     plugin_construction_ = DateTime.Now;
     Dictionary<String, ConfigNode> name_to_gravity_model = null;
-    var gravity_model_configs =
-        GameDatabase.Instance.GetConfigs(principia_gravity_model_config_name_);
-    var cartesian_configs =
-        GameDatabase.Instance.GetConfigs(principia_initial_state_config_name_);
-    ConfigNode gravity_model_config = null;
-    if (gravity_model_configs.Length == 1) {
-      gravity_model_config = gravity_model_configs[0].config;
+    ConfigNode gravity_model_config = GameDatabase.Instance.GetAtMostOneNode(
```

Comment:
Alternatively, remove `_config` from all these variables, this smells of Hungarian.

Reply:
Done.